DotNetDash/NetworkTableBackedLookup.cs

```diff
@@ -21,11 +21,13 @@ public NetworkTableBackedLookup(ITable table)
         {
             get
             {
-                return (table.GetValue(key, default(T)) is T) ? (T)(table.GetValue(key, default(T))) : default(T);
+                bool success = false;
+                var value = table.GetValue(key, NetworkTables.Value.MakeValue(default(T)));
+                return (value is T) ? value.GetValue<T>(out success) : default(T);
```

Comment:
This code does not work. GetValue now returns a Value type, therefore you cannot check if value is T, as that will never be true. PR #1 updates this to work properly with the new values, however that is waiting on some changes from FIRST before merging.